.github/workflows/ci.yml

```
@@ -51,33 +51,23 @@ jobs:
           # Don't use a 'components:' entry--we don't need them with beta/nightly, plus nightly often doesn't have them
           override: true
 
-      - name: install rustfmt+clippy on stable
-        run: |
-          rustup component add rustfmt
-          rustup component add clippy
+      - run: rustup component add rustfmt clippy
         if: contains(matrix.toolchain, 'stable')
 
-      - name: run rustfmt on stable
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
+      - run: cargo fmt
```

`--all` is a now-deprecated synonym for `--workspace`.  Without one of these this command will only run on only the `amethyst` crate and none of the other crates in the workspace.  

`-- --check` makes the command fail if it has to format something.

```suggestion
      - run: cargo fmt --workspace -- --check
```

---

Cargo fmt actually never made the switch to --workspace and has to use --all.
Can't remember the exact reasoning except they don't have --all-targets etc. to confuse it with. 

There is an old issue talking about it on their repo.

---

Yup, `--workspace` argument isn't valid with `cargo fmt`, added `--all -- --check`. 
6fcda3a

---

Yikes! Sorry for the misinformation.